packages/governance/src/index.js

```diff
@@ -17,7 +17,7 @@ export {
   validateQuestionFromCounter,
 } from './contractGovernor.js';
 
-export { handleParamGovernance } from './contractHelper.js';
+export { facetHelpers, handleParamGovernance } from './contractHelper.js';
```

Comment:
Discussed offline but explaining here for anyone else following along:
- this makes a more complete `handleParamGovernance`
- the old one was kept (because it's needed by ^) and renamed to `facetHelpers` 
- everything outside this module is using the more complete helper (`handleParamGovernance`) but this is expected so consumers have to option of getting the lighter help